# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/dbus/hotfix-tmpdir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- dbus-1.16.2/meson.build.vanilla	2025-06-28 16:48:31.542875297 +0200
+++ dbus-1.16.2/meson.build	2025-06-28 16:48:41.509057981 +0200
@@ -1056,7 +1056,7 @@
     'import os; [print(os.getenv(e, "")) for e in ["TMPDIR", "TEMP", "TMP"]]',
     check: true,
 ).stdout()
-values += '/tmp'
+values = '/tmp'
 default_socket_dir = values.strip().split('\n')[0]
 
 test_socket_dir = get_option('test_socket_dir')
